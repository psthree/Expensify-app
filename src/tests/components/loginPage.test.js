import React from 'react';
import { shallow } from 'enzyme';
import LoginPage from '../../components/LoginPage';

//snapshot test
test('Should render the login component', () => {
  const wrapper = shallow(<LoginPage />);
  expect(wrapper).toMatchSnapshot();
});
