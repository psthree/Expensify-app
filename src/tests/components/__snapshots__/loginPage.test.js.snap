// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the login component 1`] = `
<div>
  <button>
    Login
  </button>
</div>
`;

exports[`Should render the not found component 1`] = `
<div>
  <button>
    Login
  </button>
</div>
`;
